### DISCUSSION TITLE
Minimum length subtitle

### DISCUSSION THREAD

**Discussion by Vicari0** [Purfview/whisper-standalone-win#366] [Q&A] [answered]
Is it somehow possible to use a **minimum subtitle length** and a **minimum gap** between two successive subtitles?
14
00:00:57,000 --> 00:00:59,840
Dus ieder jaar een andere groente op
hetzelfde stukje grond.

15
00:01:00,180 --> 00:01:02,500
Maar ook deze planten hebben
natuurlijk...

16
00:01:02,500 --> 00:01:03,420
water nodig.

Title 16 is only 0,880 long. It would be very nice to have a minimum subtitle length of 1.20 seconds.
and a minimum gap of 0.120 seconds (3 frames) between two successive subtitles.

**Comment by Purfview** [comment, marked as answer]
>Is it somehow possible to use a minimum subtitle length and a minimum gap between two successive subtitles?

No.
